runtime/src/main/java/org/corfudb/runtime/view/LayoutView.java

```diff
@@ -126,66 +124,64 @@ public Layout prepare(long epoch, long rank)
                     return cf;
                 })
                 .toArray(CompletableFuture[]::new);
-        LayoutPrepareResponse[] acceptList;
+
+        long promises = 0L;
```

Comment:
Since these promises are generated from a java collection has is of size int, why not just use an int instead?

Reply:
Done. Thanks.